dist/index.html

```diff
@@ -16,7 +16,85 @@
+        <div id="simpleModal" class="modal">
+          <div class="modal-content">
+            <span class="closeBtn">&times;</span>
+            <table class="legend-table">
+              <tr>
+                <td>
+                  <svg width="100" height="40" viewbox="0 0 350 100">
+                    <defs>
+                      <marker id="arrowhead1" markerWidth="10" markerHeight="7" 
+                      refX="0" refY="3.5" fill="#2196f3" orient="auto">
+                      <polygon points="0 0, 10 3.5, 0 7" />
+                    </marker>
+                  </defs>
+                  <line x1="0" y1="50" x2="250" y2="50" stroke="#2196f3" 
+                  stroke-width="8" marker-end="url(#arrowhead1)" />
+                </svg>
+              </td>
+              <td>Data (stateless)</td>
+              <td>
+                <svg width="100" height="40" viewbox="0 0 350 100">
+                  <defs>
+                    <marker id="endarrow-state" markerWidth="10" markerHeight="7" 
+                    refX="0" refY="3.5" fill="red" orient="auto">
+                    <polygon points="0 0, 10 3.5, 0 7" />
+                  </marker>
+                </defs>
+                <line x1="0" y1="50" x2="250" y2="50" stroke="red" 
+                stroke-width="8" marker-end="url(#endarrow-state)" />
+              </svg>
+            </td>
+            <td>#State</td>
+          </tr>
+          <tr>
+            <td><svg width="100" height="40" viewbox="0 0 350 100">
+              <defs>
+                <marker id="arrowhead" markerWidth="10" markerHeight="7" 
+                refX="0" refY="3.5" fill="#2196f3" orient="auto">
+                <polygon points="0 0, 10 3.5, 0 7" />
+              </marker>
+            </defs>
+            <line x1="0" y1="50" x2="250" y2="50" stroke="#2196f3"
+            stroke-width="8" stroke-dasharray="30" marker-end="url(#arrowhead)" />
+          </svg>
+        </td>
+        <td>Data (stateful)</td>
+        <td>
+          <svg width="100" height="40" viewBox="0 0 350 100">
+            <defs>
+              <marker id="startarrow" markerWidth="10" markerHeight="7" 
+              refX="10" refY="3.5" orient="auto">
+              <polygon points="10 0, 10 7, 0 3.5" fill="#71c94f" />
+            </marker>
+            <marker id="endarrow" markerWidth="10" markerHeight="7" 
+            refX="0" refY="3.5" orient="auto" markerUnits="strokeWidth">
+            <polygon points="0 0, 10 3.5, 0 7" fill="#71c94f" />
+          </marker>
+        </defs>
+        <line x1="100" y1="50" x2="250" y2="50" stroke="#71c94f" stroke-width="8" 
+        marker-end="url(#endarrow)" marker-start="url(#startarrow)" />
+      </svg>
+    </td>
+    <td>Network</td>
+    </tr>
+    <tr>
+      <td><img src="file:///C:/Users/jayja/karl-ui/src/img/clock.png" width="40" height="40" style="float:center;margin:0px 20px" ></td>
+      <td>Time</td> 
+      <td><div class="modual-box"></div>
```

Comment:
Note: spelled "module"

Reply:
I fixed the spelling of the "module-box" class.